spectacles/cli.py

```diff
@@ -5,16 +5,17 @@
 import argparse
 import logging
 import os
-from typing import Callable, Iterable, List
+from typing import Callable, Iterable, List, Optional
 from spectacles import __version__
 from spectacles.runner import Runner
 from spectacles.client import LookerClient
-from spectacles.exceptions import SpectaclesException, ValidationError
+from spectacles.exceptions import SpectaclesException, ValidationError, SqlError
 from spectacles.logger import GLOBAL_LOGGER as logger, FileFormatter
 import spectacles.printer as printer
 
 LOG_FILENAME = "spectacles.log"
 LOG_FILEPATH = Path()
+LOG_DIRECTORY = Path("logs")
```

Comment:
I don't think we actually need this since it's already the default value for the `log_dir` CLI arg, which is passed to `set_file_handler`.

Reply:
The `log_failing_sql` function is going to require knowledge of either the directory it should create the 'queries' directory in, or the directory that it should dump the files themselves.

I agree having `LOG_DIRECTORY` is clunky. My suggestion is to actually declare something like `QUERY_FILEPATH` as a global variables that also gets set in `set_file_handler`. We then just pass that `QUERY_FILEPATH` to the `log_failing_sql` function. Would you prefer that approach?

Reply:
(Current code reflects this new approach if you want to have a look.)